packages/plugin-ext/src/main/browser/menus/menus-contribution-handler.ts

```diff
@@ -496,21 +497,27 @@ export class MenusContributionPointHandler {
             return Disposable.NULL;
         }
 
-        const selectedResource = () => {
+        const getSelectedResources = () => {
             const selection = this.selectionService.selection;
+            const resourceParams = [];
```

Comment:
This is currently implicitely typed as being `any[]`. Can you add an explicit type here (even if it's `any[]`)?